homeassistant/components/weather/met.py

```diff
@@ -16,59 +16,16 @@
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.event import (async_track_utc_time_change,
                                          async_call_later)
-from homeassistant.util import dt as dt_util
+import homeassistant.util.dt as dt_util
 
-REQUIREMENTS = ['pyMetno==0.2.0']
+REQUIREMENTS = ['pyMetno==0.3.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ATTRIBUTION = "Weather forecast from met.no, delivered " \
                    "by the Norwegian Meteorological Institute."
 DEFAULT_NAME = "Met.no"
 
-# https://api.met.no/weatherapi/weathericon/_/documentation/#___top
-CONDITIONS = {1: 'sunny',
-              2: 'partlycloudy',
-              3: 'partlycloudy',
-              4: 'cloudy',
-              5: 'rainy',
-              6: 'lightning-rainy',
-              7: 'snowy-rainy',
-              8: 'snowy',
-              9: 'rainy',
-              10: 'rainy',
-              11: 'lightning-rainy',
-              12: 'snowy-rainy',
-              13: 'snowy',
-              14: 'snowy',
-              15: 'fog',
-              20: 'lightning-rainy',
-              21: 'lightning-rainy',
-              22: 'lightning-rainy',
-              23: 'lightning-rainy',
-              24: 'lightning-rainy',
-              25: 'lightning-rainy',
-              26: 'lightning-rainy',
-              27: 'lightning-rainy',
-              28: 'lightning-rainy',
-              29: 'lightning-rainy',
-              30: 'lightning-rainy',
-              31: 'lightning-rainy',
-              32: 'lightning-rainy',
-              33: 'lightning-rainy',
-              34: 'lightning-rainy',
-              40: 'rainy',
-              41: 'rainy',
-              42: 'snowy-rainy',
-              43: 'snowy-rainy',
-              44: 'snowy',
-              45: 'snowy',
-              46: 'rainy',
-              47: 'snowy-rainy',
-              48: 'snowy-rainy',
-              49: 'snowy',
-              50: 'snowy',
-              }
 URL = 'https://aa015h6buqvih86i1.api.met.no/weatherapi/locationforecast/1.9/'
```

Comment:
Are there different APIs? Could this be moved to the library?

Reply:
No, `aa015h6buqvih86i1` is a special code that we have got from met.no, so they can track the load we generate on their servers.

Reply:
Cool! Thanks!